goalc/compiler/SymbolInfo.h

```diff
@@ -31,11 +39,14 @@ class SymbolInfo {
     INVALID
   };
 
-  static SymbolInfo make_global(const std::string& name, const goos::Object& defining_form) {
+  static SymbolInfo make_global(const std::string& name,
+                                const goos::Object& defining_form,
+                                const std::optional<Metadata> meta = {}) {
     SymbolInfo info;
     info.m_kind = Kind::GLOBAL_VAR;
     info.m_name = name;
     info.m_def_form = defining_form;
+    info.m_meta = meta ? meta.value() : Metadata();
```

Comment:
minor, but I think this would more normally be written as
```
if (meta) {
  info.m_meta = *meta;
}
// no need to set it to Metadata(), the default constructor of SymbolInfo does this for us.
```
or
```
info.m_meta = meta.value_or({});
```
which sets it to `meta` if it exists, or `{}` otherwise.